core/include/usChrono.h

```diff
@@ -20,12 +20,20 @@
 
 =============================================================================*/
 
-#include <usGlobalConfig.h>
 
-namespace us {
+#ifndef USCLOCK_H
```

Comment:
This file move operation is odd. Is this accidental or Github showing it wrong ? This is a new file and should have no history.